packages/kernel-5.15/kernel-5.15.spec

```
@@ -85,6 +88,13 @@ Summary: Modules for the Linux kernel
 %description modules
 %{summary}.
 
+%package mkfs-xfs-conf
+Summary: mkfs configurations for different filesystems
```

Since this is specific for XFS, the summary should call it out:

```suggestion
Summary: mkfs configurations for the XFS filesystem
```